//
//  Error Handling.swift
//  Learning RxSwift
//
//  Created by 刘嘉豪 on 2020/10/25.
//  Copyright © 2020 刘嘉豪. All rights reserved.
//

import Foundation
import RxSwift

// MARK: - Error Handling

/**
 一旦序列发生了一个error事件，整个序列将会被终止。RxSwift主要有两种错误处理机制：
 retry：重试
 catch：恢复
 */

// MARK: retry

func testRetry() {
    
}

// MARK: retryWhen

// MARK: catchError

// MARK: result
